react/src/functions.tsx

```diff
@@ -389,6 +390,24 @@ export const updateFiltersAndRequery = (tableRef: React.MutableRefObject<any>) =
     setTimeout(() => tableRef.current.onQueryChange);
 };
 
+/**
+ * Update a material-table query from outside the table
+ * Material Table has its own state that stores a query object.
+ * We want to add an extra property to this object to enable exact match query for participant and family codename.
+ */
+
+export const updateSearchTypeAndRequery = (
+    tableRef: React.MutableRefObject<any>,
+    searchTypeOptions: SearchType
+) => {
+    const oldQuery = tableRef.current.state.query;
+    if (oldQuery.searchType) {
+        tableRef.current.state.query.searchType.push(searchTypeOptions);
+    }
```

Comment:
This doesn't do anything because you are overwriting the property in the following line.